src/main.rs

```diff
@@ -49,6 +51,7 @@ fn upload(data: Data, content_type: &ContentType) -> io::Result<String> {
 #[derive(Serialize, Deserialize, Debug)]
 struct Metadata {
     content_type: String,
+    time_stamp: u64,
```

Comment:
for backwards compatability with previous versions, should be optional

```suggestion
    time_stamp: Some(u64),
```

see: https://doc.rust-lang.org/std/option/index.html